src/main/java/software/bernie/techarium/trait/ICanCopy.java

```diff
@@ -0,0 +1,7 @@
+package software.bernie.techarium.trait;
+
+public interface ICanCopy {
+    default public Object deepCopy() {
```

Comment:
I actually already implemented this in my multiblock branch, but just override the native clone() method, this is unnecessary